### ISSUE TITLE
Filtering with an __in query and then using `values_list` returns nothing

### ISSUE THREAD

**Issue by moggers87** [potatolondon/djangae#470]
The follow query returns no results if `some_model_pks` contains more than one value:

```
SomeModel.objects.filter(pk__in=some_model_pks).values_list("pk", "some_field").order_by("some_field")
```

**Comment by moggers87** [comment]
`some_field` is a charfield in this case, by the way.

**Comment by moggers87** [comment]
Looks like this has been fixed by 79318311bb73a6306749c161f3d4f62d70fe1a70
